python/tk_desktop/desktop_window.py

```diff
@@ -40,6 +42,7 @@
 from .project_commands_model import ProjectCommandModel
 from .project_commands_model import ProjectCommandProxyModel
 from .project_commands_widget import ProjectCommandDelegate
+from shotgun_desktop.desktop_login import get_shotgun_authenticator
```

Comment:
So this imports code from inside of the desktop executable python repo. I looked at that code and it is not really obvious that this method is part of a public API that can be accessed by external things. Does the desktop-internal API have a "public API" at this point already that we can add `get_shotgun_authenticator` to? I am worried that things will break because that method is changed in desktop-internal and it is not obvious that it is called from somewhere else! I think it could be useful to clarify this - maybe by establishing a "public API" in desktop internal. What do you think?